src/main/java/com/liferay/faces/util/context/FacesContextHelperUtil.java

```diff
@@ -26,14 +27,16 @@
 import javax.faces.context.FacesContext;
 import javax.faces.event.PhaseListener;
 
+import com.liferay.faces.util.client.Script;
+
 
 /**
  * This class contains a set of static convenience methods that delegate to the corresponding methods on {@link
  * FacesContextHelper}.
  *
  * @author  Neil Griffin
  */
-public class FacesContextHelperUtil {
+public final class FacesContextHelperUtil {
```

Comment:
@stiemannkj1 We should probably add the private constructor too, like we did in BridgeUtil.java in the FacesBridge API.

Reply:
@ngriffin7a, I didn't wanna do that cause it would be a major breaking change (removal of a constructor).